specs/iota-rs-ENGINEERING-SPEC-0000.md

```diff
@@ -253,63 +254,6 @@ Following are the steps for implementing this method:
 * Perform proof-of-work;
 * Store messages on the tangle using [`post_messages()`](#post_messages-post-messages);
 
-## `get_message()`
```

Comment:
Should probably stay in the higher level since this is a common thing to do, retrieving a single message by message id.